<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>207</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mass">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SelectPlanet">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать планету</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConfChang">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердить изменения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_item">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить новую планету</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_item">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить планету</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_YearOpen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_WhoOpen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EnterPlanet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sizeVictor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать количество планет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mass_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>ИДЕНТИФИКАТОР</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_YearOpen_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>ГОД ОТКРЫТИЯ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>ИМЯ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_WhoOpen_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>КТО ОТКРЫЛ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущая</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Следующая</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="Merge_in_file"/>
    <addaction name="Merge_from_file"/>
   </widget>
   <widget class="QMenu" name="about_of">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="Author"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="about_of"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="Merge_in_file"/>
   <addaction name="Merge_from_file"/>
   <addaction name="Author"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/Иконки/Записать в файл.jpg</normaloff>:/rec/Иконки/Записать в файл.jpg</iconset>
   </property>
   <property name="text">
    <string>Сохранить в файл </string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/Иконки/Взять из файла.png</normaloff>:/rec/Иконки/Взять из файла.png</iconset>
   </property>
   <property name="text">
    <string>Открыть из файла</string>
   </property>
  </action>
  <action name="Author">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/Иконки/o-nas.png</normaloff>:/rec/Иконки/o-nas.png</iconset>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </action>
  <action name="Merge_in_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/Иконки/Слияние в программе.png</normaloff>:/rec/Иконки/Слияние в программе.png</iconset>
   </property>
   <property name="text">
    <string>Добавить в файл</string>
   </property>
   <property name="toolTip">
    <string>Объединить объекты из программы с объектами в файле</string>
   </property>
  </action>
  <action name="Merge_from_file">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/rec/Иконки/Слияние в файле.png</normaloff>:/rec/Иконки/Слияние в файле.png</iconset>
   </property>
   <property name="text">
    <string>Объединить из файла</string>
   </property>
   <property name="toolTip">
    <string>Объединить объекты из файла с объектами в программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
